### PR TITLE
Packit shouldn't change changelog for Anaconda

### PR DESCRIPTION
**Pull request by jkonecny12**
We want our upstream changelog instead. It's rewriting history on the downstream but *SHHH* don't tell anyone.

### CODE REVIEW

**Review by poncovka — APPROVED** [review]
Looks good to me.

**Comment by jkonecny12** [comment]
There's a question if we want to merge this before release or if we want to test this after the release locally before merge. What do you think @M4rtinK ?

**Comment by jkonecny12** [comment]
Rebased to test tests on HEAD.

**Review by poncovka — APPROVED** [review]
LGTM

**Comment by jkonecny12** [comment]
This will not be merged probably. It's basically replaced by https://github.com/packit/packit/pull/1004.

**Comment by VladimirSlavik** [comment]
@jkonecny12 can this be closed?

**Pull request closed without merging**
